Code/GraphMol/FileParsers/ForwardSDMolSupplier.cpp

```diff
@@ -212,7 +210,7 @@ ROMol *ForwardSDMolSupplier::_next() {
     }
     d_line = line;
     if (res) {
-      this->readMolProps(res);
+      this->readMolProps(res.get());
```

Comment:
Suggestion to change the protected readMolProps(ROMol*) to

```
readMolProps(ROMol &m)
```

since m can never be null when readMolProps is called.

This changes the usage to

```
  this->readMolProps(*res);
```